.github/workflows/draft-release.yml

```
@@ -0,0 +1,28 @@
+          - major
+        default: patch
+
+jobs:
+  draft-release:
+    name: Draft SDK Release (placeholder)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Echo inputs
+        env:
+          BUMP_TYPE: ${{ inputs.bump_type }}
+          SDK: ${{ inputs.sdk }}
```

actionlint (1.7.12)

[error] property "sdk" is not defined in object type {bump_type: string}

(expression)